veg2hab/vegkartering.py

```diff
@@ -944,6 +991,110 @@ def apply_wwl(
             wwl.toevoegen_VvN_aan_List_VegTypeInfo
         )
 
+    @staticmethod
+    def _vegtypeinfo_to_multi_col(vegtypeinfos: List[VegTypeInfo]) -> pd.Series:
```

Comment:
Deze functies op de een of andere manier samen voegen met de functies de de uiteindelijke output verzorgen zou wel handig zijn, maar je loopt wel het risico dat als er verzoekjes komen voor verandering van de output we het alsnog weer los moeten trekken...... Ik heb geen echt sterke voorkeur een van beide kanten op.

Reply:
agree